### DISCUSSION TITLE
May somebody email me at my email to tell me how to use Interstellar Astro (The Code?) Thank You

### DISCUSSION THREAD

**Discussion by PrestonSun** [UseInterstellar/Interstellar#901] [Q&A] [answered]
May somebody email me at my email to tell me how to use Interstellar Astro (The Code?) Thank You

**Comment by abdirahmaneagleridgemiddleschool** [comment]
the code is npm run start

**Comment by lakshinhemachandran** [comment, marked as answer]
No he's asking the whole instructions.
just check the README file,
It should have everything
if I doesn't say it, DM someone on Discord
